src/zope/container/contained.py

```
@@ -827,9 +824,9 @@ def chooseName(self, name, object):
         # convert to unicode and remove characters that checkName does not allow
         if isinstance(name, bytes):
             name = name.decode()
-        if not isinstance(name, unicode):
+        if not isinstance(name, text_type):
             try:
-                name = unicode(name)
+                name = text_type(name)
             except:
```

Ugh, bare except, really?